website/www/site/content/en/documentation/io/built-in/snowflake.md

```diff
@@ -213,8 +213,8 @@ To pass Pipeline options via the command line, use `-DintegrationTestPipelineOpt
       Example: --database=TEST_DATABASE
   "--storageIntegrationName=<SNOWFLAKE STORAGE INTEGRATION NAME>",
       Example: --storageIntegrationName=my_integration
-  "--stagingBucketName=<GCS BUCKET NAME>",
-      Example: --stagingBucketName=gs://my_gcp_bucket
+  "--stagingBucketName=<GCS OR S3 BUCKET NAME>",
+      Example: --stagingBucketName=<gs or s3>://my_gcp_bucket
```

Comment:
nit: this still references gcp, it should probably be `my_bucket`. It'd be nice if all of the example bucket names were the same.

I'd also make it `{gs,s3}://my_bucket` to be a little more concise, but I'm fine with it as-is as well.

Reply:
Thanks @TheNeuralBit 

I change everywhere to `bucket` + introduce `{gs,s3}`